client/my-sites/preview/main.jsx

```diff
@@ -30,10 +30,28 @@ class PreviewMain extends React.Component {
 
 	state = {
 		previewUrl: null,
+		showingClose: false,
 	};
 
 	componentWillMount() {
 		this.updateUrl();
+		this.updateLayout();
+	}
+
+	updateLayout = () => {
+		this.setState( {
+			showingClose: isWithinBreakpoint( '<660px' ),
+		} );
+	}
+
+	throttledUpdateLayout = throttle( this.updateLayout, 100 );
+
+	componentDidMount() {
+		global.window && global.window.addEventListener( 'resize', this.throttledUpdateLayout );
```

Comment:
exactly. without the check here (alternatively using `typeof window !== 'undefined'`) we are preventing the `ReferenceError` that would occur when accessing non-existent variable